File: 8-linked-lists1/1-intro/main.py
# Linked Lists Introduction - Python Example

class Node:
    def __init__(self, data):
        self.data = data
        self.next = None

# Creating linked list with two nodes
node1 = Node(1)
node2 = Node(2)
node1.next = node2

# Traversing linked list
current = node1
while current:
    print(current.data)
    current = current.next

File: 8-linked-lists1/1-intro/main.java
// Linked List Introduction - Java Example

class Node {
    int data;
    Node next;
    Node(int data) {
        this.data = data;
        this.next = null;
    }
}

class LinkedListIntro {
    public static void main(String[] args) {
        Node node1 = new Node(1);
        Node node2 = new Node(2);
        node1.next = node2;

        Node current = node1;
        while (current != null) {
            System.out.println(current.data);
            current = current.next;
        }
    }
}
